api/swagger.yaml

```diff
@@ -77,43 +90,337 @@ tags:
+      | :------: | :-----: | :------: | :---: | :---: | :---: | :--------: | :----: | :---: |
+      |          |         |          | ●     |       |       |            |        |       |
+
+
+      ### Sensor coupling
+
+      A `Sensor` can be coupled to another entity. It is also possible to decouple `Sensor`s, to facilitate portable sensors that are temporarily
+      used. To couple and decouple sensors, use one of the following event types:
+      - `coupleSensorEvent`
+      - `decoupleSensorEvent`
+      The entity the `Sensor` is coupled to, or decoupled from, should be in the `involvedObjects` list of the event.
+
+      The meaning of a sensor coupling event might be a bit different in different life cycle phases:
+      - In the `planned` and `projected` phases, the event should be interpreted as "it is planned (or projected) that this sensor will be coupled
+        to this entity. This might be useful if you want to express a planning in OpenTripModel and you want to explicitely document the action
+        of placing a portable sensor in e.g. a trailer.
+      - In the `actual` life cycle phase, the event can represent the moment a sensor is installed. But sometimes a sensor is permanently installed,
+        and you just need a way to tell the system that a certain sensor is associated with a certain vehicle or physical location. In any case, a
+        system can use the couple events in the `actual` life cycle phase to know that update events to the sensor also apply to the associated
+        entity.
+      - In the `realized` life cycle phase, the couple and decouple events might be important for monitoring and auditing. E.g. once you know a
+        temperature sensor is coupled to a trailer, you can use it to create an audit trail of the temperature inside the trailer during a trip.
+
+      #### Applicable entities
+
+      | Location | Vehicle | Shipment | Trip  | Route | Actor | Constraint | Sensor | Event |
+      | :------: | :-----: | :------: | :---: | :---: | :---: | :--------: | :----: | :---: |
+      | ●        | ●       | ●        |       |       | ●     |            | ●      |       |
+
+      ## Meta events
+
+      Meta events are events about events. There are two types of meta events: `cancelEvent` and `ignoreEvent`. As a rule of thumb, a `cancelEvent`
+      is to be used for events that you created yourself and want to cancel (or replace), while an `ignoreEvent` is meant to notify that an event
+      sent by another system or actor is to be ignored. An example of the latter might be an event to ingore a `sensorUpdateEvent`, because the
+      sensor is defective.
+
+      ### `cancelEvent`
+
+      Cancels one or more `Event`. Since events can only be created and not deleted nor updated, a `cancelEvent` is needed when you need to change
+      or remove an event. To be canceled events should be set as `involvedObjects`. Note that you can't cancel events in another lifecycle phase.
```

Comment:
Maybe elaborate a bit more about the lifecycle phase? I.e.: a cancel event can only cancel events in its own lifecycle phase.